docs/user_guide/method/method.rst

```diff
@@ -0,0 +1,2384 @@
+      \vdots & \vdots &  & \vdots
+      \end{array}
+
+    Below is an energy level diagram of two coupled spin :math:`I=1/2` nuclei with
+    transition labeled according to their transition symmetry function values.  Note
+    that each transition has a unique set of transition symmetry function values.
+
+    .. figure:: ../../_static/CoupledOneHalf.*
+        :width: 650
+        :alt: figure
+        :align: center
+
+
+Single-Spin Queries
+-------------------
+
+Based on the review above, we now know that the spin :math:`I=1`, the transition
+:math:`\ketbra{-1}{0}` can be selected with :math:`{(\text{p}_I,\text{d}_I) =
+(-1,1)}`.  In **mrsimulator**, this transition is selected during a
+**SpectralEvent** using the **SymmetryQuery** and **TransitionQuery** objects,
+as defined in the code below.
+
+.. plot::
+    :context: reset
+
+    from mrsimulator.method.query import SymmetryQuery, TransitionQuery
+    from mrsimulator.method import SpectralEvent
+
+    symm_query = SymmetryQuery(P=[-1], D=[1])
+    trans_query = TransitionQuery(ch1=symm_query)
+    event = SpectralEvent(fraction=1, transition_query=[trans_query])
+
+.. note::
+    Python dictionaries can also be used to create and initialize an object. To do this,
+    the dictionary must use the object's attribute names as the key strings. For
+    example, the code above could have been written
+
+    .. plot::
+        :context: close-figs
+
+            symm_query = {"P":[-1], "D":[1]}
+            trans_query = {"ch1":symm_query}
+            event = {"fraction":1, "transition_query":[trans_query]}
+
+In the example above, the **SymmetryQuery** object is created and assigned to
+the **TransitionQuery** attribute ``ch1``, i.e., it acts on the isotope in the
+"first channel". Recall that the ``channels`` attribute of the **Method** object
+holds an ordered list of isotope strings. This list's first, second, and third
+isotopes are associated with ``ch1``, ``ch2``, and ``ch3``, respectively.
+Currently, **mrsimulator** only supports up to three channels, although this may
+be increased in future versions.
+
+The **TransitionQuery** object goes into an unordered list in the
+``transition_queries`` attribute of a **SpectralEvent** object. The
+**SpectralEvent** object, in turn, is added to an ordered list in the ``events``
+attribute of a **SpectralDimension** object. All this is illustrated in the code
+below.
+
+
+.. plot::
+    :context: close-figs
+
+    from mrsimulator import Site, Coupling, SpinSystem, Simulator
+    from mrsimulator import Method, SpectralDimension
+    from mrsimulator import signal_processor as sp
+    import matplotlib.pyplot as plt
+    import numpy as np
+
+    # Create Single Site and Spin System
+    deuterium = Site(
+      isotope="2H",
+      isotropic_chemical_shift=10,  # in ppm
+      shielding_symmetric={"zeta":-80, "eta":0.25},  # zeta in ppm
+      quadrupolar={"Cq":10e3, "eta":0.0,"alpha":0, "beta":np.pi/2, "gamma":0}
```

Comment:
Space in-between colon and dict value

Reply:
I thought the rule was no space?

Reply:
According to pep8, there is a space. 

I'm commenting on these things so I remember to come back to them later. I can also run the code blocks through a linter on my machine.